dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsolute.java

```diff
@@ -99,15 +101,24 @@ private String replaceAllContainerRelativePath(
     }
 
     private List<String> getRelativePaths(final String drawedBody) {
-        final TemplateLayout templateLayoutFromJSON = DotTemplateTool.getTemplateLayout(drawedBody);
+        final Set<ContainerUUID> templateContainers = DotTemplateTool.getTemplateContainers(
+                drawedBody);
 
-        return templateLayoutFromJSON.getContainersIdentifierOrPath()
+        return getContainersIdentifierOrPath(templateContainers)
                 .stream()
                 .filter((String idOrPath) -> FileAssetContainerUtil.getInstance().isFolderAssetContainerId(idOrPath))
                 .filter((String idOrPath) -> !FileAssetContainerUtil.getInstance().isFullPath(idOrPath))
                 .collect(Collectors.toList());
     }
 
+    private static Set<String> getContainersIdentifierOrPath(final Set<ContainerUUID> containerUUIDS) {
+
+        return containerUUIDS
+                .stream()
+                .map(ContainerUUID::getIdentifier)
+                .collect(Collectors.toSet());
```

Comment:
There is an unnecessary call to `collect()` since on line 108 is going back to a stream. Basically I would remove this method and copy-paste the its body, except for the termination method, to the calling method (`getRelativePaths()(`)

Reply:
done https://github.com/dotCMS/core/pull/22604/commits/e0ffb54258b3a929d89c7089ffca177a18823ea0#diff-b8921f877ae173e9ff875d93630f0f026b4b96f3d23e0a98d3b768208be93399R119